stories/Input.stories.ts

```diff
@@ -20,45 +22,54 @@ const Template: Story<InputComponent> = (args: InputComponent) => ({
 });
 
 export const Input = Template.bind({});
-Input.args = {};
+Input.args = {
+  valueChange: action('valueChange'),
+};
 
 export const IconLeft = Template.bind({});
 IconLeft.args = {
   iconInput: 'pencil',
   iconDirection: 'left',
+  valueChange: action('valueChange'),
```

Comment:
I believe that you can add only on template, like a SimpleMenu.stories. That way you need add only here.

![image](https://user-images.githubusercontent.com/6165180/203801954-6dfae922-398b-4a65-b96d-28077aac4b6c.png)

Reply:
nice!